tinygrad/renderer/cstyle.py

```diff
@@ -173,7 +173,8 @@ def ssa(prefix:str, u:Optional[UOp]=None):
         elif uop is UOps.DEFINE_GLOBAL:
           bufs[u] = (nm:=f"data{args}", (dtype, False))
           r[u] = nm
-        elif uop is UOps.WMMA: kk(f"{self.render_dtype(dtype)} {ssa('wmma',u)} = __{args[0]}({r[src[0]]}, {r[src[1]]}, {r[src[2]]});")
+        elif uop is UOps.WMMA:
+          kk(f"{self.render_dtype(dtype)} {ssa('wmma',u)} = __{args[0]}({r[src[0]]}, {r[src[1]]}{'' if self.device == 'CLANG' else f', {r[src[2]]}'});") # noqa:E501
```

Comment:
I'm not sure this is right. Does it accumulate?

Reply:
I was thinking that for making this ultimately correct, the accs should be rendered as well as for the other TC backend and the data should be loaded into the AMX accumulation registers. This will imply even more speed reduction and I don't see them being used in the short term due to the lack of folding mentioned above but is the way without hacks. If you want I can do that tommorrow.

Reply:
moved the `test_tensor_core_opts` split into separate PR